Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM node:boron
+MAINTAINER Afroz Ahamad <enigmaeth@gmail.com>
+
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+
+RUN apt-get update
+
+# install deps
+RUN apt-get install -y --no-install-recommends build-essential python-dev libpq-dev libevent-dev libmagic-dev python-pip && apt-get clean -y && curl -sL https://deb.nodesource.com/setup_4.x | bash && apt-get install -y --force-yes nodejs
```

Comment:
Why are you installing nodejs ? The `node:boron` image comes with Node.js pre-installed. Please use the same.